nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -215,6 +237,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
         self.megatron_amp_O2 = cfg.get('megatron_amp_O2', False)
 
         self.mcore_gpt = cfg.get('mcore_gpt', False)
+        self.spec_name = cfg.get('name', '')
```

Comment:
I don't think `name` and `spec_name` are the same thing?

Reply:
@HuiyingLi ,this was originally due to we want to use falcon spec right?